SU-HSDO/suhumsci
# HSD8-536 Accordion pattern and paragraph type

# READY FOR REVIEW

# Summary
- Accordion pattern and paragraph type
- uses drupal core collapse javascript for accessibility and IE help.

# Needed By (Date)
- End of the sprint. 2/13

# Urgency
- Low

# Steps to Test

1. Checkout this branch
1. sync a site `blt drupal:sync --site=[sitename]`
1. create a basic page and add an accordion component
1. ensure it functions as expected.


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
